src/main/java/de/teamlapen/vampirism/player/vampire/BloodStats.java

```diff
@@ -41,12 +42,17 @@ public int addBlood(int amount, float saturationModifier) {
 
     @Override
     public boolean consumeBlood(int a) {
```

Comment:
return true everytime?

Reply:
:see_no_evil: good point

Reply:
but no need, to create a new method, just fix the old one :D